chunky/src/java/se/llbit/chunky/entity/BeaconBeam.java

```diff
@@ -48,13 +54,12 @@
   private final JsonObject pose;
   private double scale = 1;
   private int height = 256;
-  private Map<Integer, BeaconBeamMaterial> materials = new HashMap<>();
+  private final Map<Integer, BeaconBeamMaterial> materials = new HashMap<>();
```

Comment:
_fastutils_ has a better map for this: http://fastutil.di.unimi.it/docs/it/unimi/dsi/fastutil/ints/Int2ObjectOpenHashMap.html

But better ask @NotStirred, he knows this stuff better than me

Reply:
Yeah having a boxed `int` to `Object` map is never gonna be as fast as the fastutils `Int2ObjectOpenHashMap`